Brokerages/Alpaca/AlpacaBrokerage.Utility.cs

```diff
@@ -0,0 +1,428 @@
+        /// </summary>
+        private Order ConvertOrder(IOrder order)
+        {
+            var type = order.OrderType;
+
+            Order qcOrder;
+            switch (type)
+            {
+                case Markets.OrderType.Stop:
+                    qcOrder = new StopMarketOrder
+                    {
+                        StopPrice = order.StopPrice.Value
+                    };
+                    break;
+
+                case Markets.OrderType.Limit:
+                    qcOrder = new LimitOrder
+                    {
+                        LimitPrice = order.LimitPrice.Value
+                    };
+                    break;
+
+                case Markets.OrderType.StopLimit:
+                    qcOrder = new StopLimitOrder
+                    {
+                        Price = order.StopPrice.Value,
+                        LimitPrice = order.LimitPrice.Value
+                    };
+                    break;
+
+                case Markets.OrderType.Market:
+                    qcOrder = new MarketOrder();
+                    break;
+
+                default:
+                    throw new NotSupportedException(
+                        "An existing " + type + " working order was found and is currently unsupported. Please manually cancel the order before restarting the algorithm.");
+            }
+
+            var instrument = order.Symbol;
+            var id = order.OrderId.ToString();
+
+            qcOrder.Symbol = Symbol.Create(instrument, SecurityType.Equity, Market.USA);
+
+            if (order.SubmittedAt != null)
+            {
+                qcOrder.Time = order.SubmittedAt.Value;
+            }
+
+            qcOrder.Quantity = order.Quantity;
+            qcOrder.Status = Orders.OrderStatus.None;
+            qcOrder.BrokerId.Add(id);
+
+            Order orderByBrokerageId;
+            lock (_locker)
+            {
+                orderByBrokerageId = _orderProvider.GetOrderByBrokerageId(id);
+            }
+
+            if (orderByBrokerageId != null)
+            {
+                qcOrder.Id = orderByBrokerageId.Id;
+            }
+
+            if (order.ExpiredAt != null)
+            {
+                qcOrder.Properties.TimeInForce = Orders.TimeInForce.GoodTilDate(order.ExpiredAt.Value);
+            }
+
+            return qcOrder;
+        }
+
+        /// <summary>
+        /// Converts an Alpaca position into a LEAN holding.
+        /// </summary>
+        private static Holding ConvertHolding(IPosition position)
+        {
+            const SecurityType securityType = SecurityType.Equity;
+            var symbol = Symbol.Create(position.Symbol, securityType, Market.USA);
+
+            return new Holding
+            {
+                Symbol = symbol,
+                Type = securityType,
+                AveragePrice = position.AverageEntryPrice,
+                ConversionRate = 1.0m,
+                CurrencySymbol = "$",
+                Quantity = position.Quantity
```

Comment:
Should the same logic be applied here? I think it might not be necessary today since `Alpaca` does not support shorting